scripts/gulp/vendor-bundles.js

```diff
@@ -18,10 +17,10 @@ module.exports = {
   //
   // Modules may be excluded from parsing for two reasons:
   //
-  // 1. The module is large (eg. jQuery) and contains no require statements,
+  // 1. The module is large and contains no require statements,
   //    so skipping parsing speeds up the build process.
   // 2. The module is itself a compiled Browserify bundle containing
   //    internal require() statements, which should not be processed
   //    when including the bundle in another project.
-  noParseModules: ['jquery'],
+  noParseModules: [],
```

Comment:
Did you leave this here as a useful hook for later, potentially?

Reply:
Yes, exactly.